### PR TITLE
Redraw CivilopediaScreen

### PR DESCRIPTION
**Pull request by lishaoxia1985**
XueR

### CODE REVIEW

File: core/src/com/unciv/ui/CivilopediaScreen.kt [hunk 1]
```diff
@@ -59,18 +63,18 @@ class CivilopediaScreen(ruleset: Ruleset) : CameraStageBaseScreen() {
                 .filter { "Will not be displayed in Civilopedia" !in it.uniques && !(it.isWonder || it.isNationalWonder) }
                 .map {
                     CivilopediaEntry(it.name, it.getDescription(false, null, ruleset),
-                            ImageGetter.getConstructionImage(it.name).surroundWithCircle(50f))
+                            ImageGetter.getConstructionImage(it.name).surroundWithCircle(54f))
```
Comment (yairm210) [inline core/src/com/unciv/ui/CivilopediaScreen.kt:66]: Please change all these sizes to a variable

File: core/src/com/unciv/ui/CivilopediaScreen.kt [hunk 2]
```diff
@@ -36,19 +37,22 @@ class CivilopediaScreen(ruleset: Ruleset) : CameraStageBaseScreen() {
...
+            }
             if (entry.image != null)
                 if (category == "Terrains")
-                    entryButton.add(entry.image).padRight(24f)
+                    entryButton.add(entry.image).padLeft(20f).padRight(10f)
                 else
-                    entryButton.add(entry.image).size(50f).padRight(10f)
-            entryButton.add(entry.name.toLabel())
+                    entryButton.add(entry.image).padLeft(10f)
+            entryButton.add(entry.name.toLabel(Color.WHITE, 25)).pad(10f).expandX().fillX()
```
Comment (yairm210) [inline core/src/com/unciv/ui/CivilopediaScreen.kt:49]: I don't think both expandX and fillX are necessary?

File: core/src/com/unciv/ui/utils/ImageGetter.kt [hunk 3]
```diff
@@ -241,18 +240,20 @@ object ImageGetter {
...
                 starTable.centerX(circle)
-                starTable.y = 5f
+                starTable.y = size / 6f
                 circle.addActor(starTable)
             }
             return circle
         }
-        return getImage("UnitPromotionIcons/" + promotionName.replace(' ', '_') + "_(Civ5)")
+        return Table().apply {
+            add(getImage("UnitPromotionIcons/" + promotionName.replace(' ', '_') + "_(Civ5)")).size(size)
```
Comment (yairm210) [inline core/src/com/unciv/ui/utils/ImageGetter.kt:255]: This is no longer needed, it was for a transition period where we were converting the file names.
We can just assume that the image exists and worst case, we'll get a white image

**Comment by lishaoxia1985** [comment]
@yairm210 now it's ok, although the size of terrain icon is the same as the other icon's size, their size looks different.

**Pull request merged**